messages/en.json

```
@@ -70,5 +70,16 @@
   "admin/editor.productSummaryList.installmentCriteria.max-without-interest": "Maximum without interest",
   "admin/editor.productSummaryList.installmentCriteria.max-with-interest": "Maximum",
   "admin/editor.productSummaryList.analyticsListName.title": "List name in Analytics",
-  "admin/editor.productSummary.trackListName.title": "Should track list name"
+  "admin/editor.productSummary.trackListName.title": "Should track list name",
```

```suggestion
  "admin/editor.productSummary.trackListName.title": "Should track list name",
```  >>>>> "admin/editor.productSummary.trackListName.title": "Track list name",